utils/build-toolchain

```diff
@@ -0,0 +1,50 @@
+#/bin/bash
+#
+# utils/build-toolchain - documents process for building a toolchain
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+cd "$(dirname $0)/.."
+SRC_DIR=$PWD
+
+YEAR=$(date +"%Y")
+MONTH=$(date +"%m")
+DAY=$(date +"%d")
+TOOLCHAIN_VERSION="swift-SNAPSHOT-${YEAR}-${MONTH}-${DAY}-a"
+ARCHIVE="${TOOLCHAIN_VERSION}-osx.tar.gz"
+SYM_ARCHIVE="${TOOLCHAIN_VERSION}-osx-symbols.tar.gz"
+BUNDLE_IDENTIFIER="org.swift.${YEAR}${MONTH}${DAY}"
```

Comment:
I don't think we should encourage other developers to build toolchain with the "org.swift" identifier.  Only the official CI should be using it.  This part should be a command-line parameter.